src/views/CohortPage.js

```diff
@@ -1,20 +1,23 @@
-import React from "react";
-import { Title } from "../components/Title/Title";
-import { Sidebar } from "../components/Sidebar/Sidebar";
-import { BarGraph } from "../components/Graphs/BarGraph";
-import { CohortPageStyle } from "./CohortPageStyle";
-import { GraphSectionStyle } from "./GraphSectionStyle";
+import React from 'react';
+import { Title } from '../components/Title/Title';
+import { Sidebar } from '../components/Sidebar/Sidebar';
+import { BarGraph } from '../components/Graphs/BarGraph';
+import { CohortPageStyle } from './CohortPageStyle';
+import { GraphSectionStyle } from './GraphSectionStyle';
+import { Wrapper } from '../GlobalStyle';
 
 //import "../views/CohortPage.scss";
 
 export const CohortPage = props => {
   return (
-    <CohortPageStyle>
-      <Sidebar />
-      <GraphSectionStyle>
-        <Title />
-        <BarGraph />
-      </GraphSectionStyle>
+    <CohortPageStyle grid>
+      <Wrapper>
+        <Sidebar />
+        <GraphSectionStyle>
+          <Title />
```

Comment:
Hi @yuliwna1, I noticed that the <Title /> and <BarGraph /> components no longer render on the CohortPage in this branch.